AlembicImporter/Packages/com.unity.formats.alembic/Documentation/com.unity.formats.alembic.md

```diff
@@ -79,53 +80,65 @@ The Alembic exporter can be customized by setting the properties on this compone
+|__Ignore First Frame__ |If enabled, do not capture first frame. Only available if **Capture On Start** enabled|
+|__Max Capture Frame__ |The frame to stop capturing at|
+|__Detailed Log__ |Provide detailed Debug logs of each frame that is captured|
+|__Begin Recording__ |In play mode, click button to begin recording. Use if **Capture On Start** is disabled|
 |__One Shot__|Button to export the current frame to the ABC file.|
 
+> ***Note:*** The current capture target object is determined at the start of capture, and does not change in the middle. Enabling or disabling the object does not affect the capture.
+> Also, any objects generated after the start of the capture are not captured.
+
+> ***Note:*** Be careful when deleting the target object in the middle of capture. In this case, the capture of that object will be interrupted, and the Alembic file may have a non-uniform number of samples as a result. Some software may not handle this properly.
+
+> ***Note:*** An ID (e.g. "(0000283C)") is given to the name of the Alembic node on export. This is a measure to avoid name conflicts as Alembic has a rule that each node in a hierarchy should have a unique name. 
+
+
+## Known Issues
+
+* Material export is currently not supported
+
 # Controlling Alembic playback
 
 The import and playback of Alembic data is controlled by the `Alembic Stream Player` component.
 
-If you change the Time parameter you can see that Mesh animate. To play the animation this parameter can be controlled from the Timeline, Animator component or via scripts.
+If you change the Time parameter you can see the Mesh animate. To play the animation this parameter can be controlled from the Timeline, Animator component or via scripts.
 
 Vertex Motion Scale is a magnification factor when calculating velocity. The greater the velocity and motion scale, the more blurring will be applied by the MotionBlur post processing effect.
 
-You can customized import and playback through the properties on this component:
+You can customize import and playback through the properties on this component:
 
 ![The Stream Player Settings window](images/abc_stream_player.png)
 
 | Property:| Function: |
 |:---|:---| 
-|__Time Range__ | {TODO}|
-|__seconds__ | {TODO}|
-|__Time__ | {TODO}|
-|__Vertex Motion Scale__ | {TODO}|
-|__Ignore Visibility__ |{TODO} |
-|__Async Load__ |{TODO} |
-|__Recreate Missing Nodes__ |{TODO} |
+|__Time Range__ |Select the range of the imported animation in seconds to be able to play back. By default this is the entire animation.|
+|__Time__ |The time in seconds of the animation that is currently displayed on the screen. Scrub or animate this parameter to play the animation. Range of values is from 0 to animation length. |
+|__Vertex Motion Scale__ | Set the magnification factor when calculating velocity. Greater velocity means more blurring when used with MotionBlur.|
+|__Ignore Visibility__ |Ignore import of animated visibility from the ABC file when playing|
```

Comment:
not too sure about this setting and the two below it

Reply:
@alexismorin , can you clarify that for us (what do the last 3 options really do)?